mmaction/datasets/pipelines/formating.py

```diff
@@ -371,3 +372,53 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += f"(input_format='{self.input_format}')"
         return repr_str
+
+
+@PIPELINES.register_module()
+class FormatGCNInput:
+    """Format final skeleton shape to the given input_format.
+
+    Required keys are "keypoint" and "keypoint_score", added or modified
+    keys are "keypoint" and "input_shape".
+
+    Args:
+        input_format (str): Define the final skeleton format.
+    """
+
+    def __init__(self, input_format, num_person=2):
+        self.input_format = input_format
+        if self.input_format not in ['NCTVM']:
+            raise ValueError(
+                f'The input format {self.input_format} is invalid.')
+        self.num_person = num_person
```

Comment:
`num_person` -> `num_persons`

Reply:
I found that no variable named num_persons in the project, but found several variables named num_person. Thus I think no need to modify it.

Reply:
ok